### PR TITLE
Initial code to pull artifacts from S3 bucket and CLI Tool

### PR DESCRIPTION
**Pull request by varkha-d-sharma**

### CODE REVIEW

File: cmflib/cli/parser.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import argparse
+import logging
+import os
+import sys
+
+from cmflib.cli import CmfParserError
+from cmflib.commands import (
```
Comment (annmary-roy) [inline cmflib/cli/parser.py:7]: Please check, I think the folder is name 'command' not 'commands'

File: cmflib/minio_artifacts.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        secret_key = ""
+        try:
+            endpoint, access_key, secret_key = self.get_dvc_config(current_directory)
+            client = Minio(
+                endpoint,
+                access_key=access_key,
+                secret_key=secret_key,
+                secure=False
+            )
+            found = client.bucket_exists("dvc-art")
```
Comment (annmary-roy) [inline cmflib/minio_artifacts.py:38]: bucket name should not be assumed , the user can have different bucket name

File: cmflib/cli/parser.py (hunk 1 above)
Comment (varkha-d-sharma) [reply cmflib/cli/parser.py:7]: Made the change

File: cmflib/minio_artifacts.py (hunk 2 above)
Comment (varkha-d-sharma) [reply cmflib/minio_artifacts.py:38]: Made the required change

**Pull request merged**